paparazzi/src/main/java/app/cash/paparazzi/DeviceConfig.kt

```diff
@@ -66,9 +67,11 @@ data class DeviceConfig(
   val xdpi: Int = 320,
   val ydpi: Int = 320,
   val orientation: ScreenOrientation = ScreenOrientation.PORTRAIT,
-  val nightMode: NightMode = NOTNIGHT,
   val density: Density = Density.XHIGH,
   val fontScale: Float = 1f,
+  val layoutDirection: LayoutDirection? = null,
+  val locale: LocaleQualifier = LocaleQualifier(),
```

Comment:
Instead of exposing the internal qualifier, let's expose `locale: String` and map internally?  Looking at AOSP, there's a Locale type, that we can eventually port that too, but it's in a later version of sdk-common than we can support right now due to our resource loading mechanism (explained in https://github.com/cashapp/paparazzi/issues/524).

Reply:
Replaced it with a nullable string. Can make it non-nullable and check for empty/blank string instead. As a second option, we can introduce `value class  Locale(tag: String)` for a better type reference in tests.

Reply:
Replaced with non-nullable `locale: String` here 817420e